hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanner.java

```diff
@@ -187,9 +187,18 @@ private List<String> getPartitionPathsForIncrementalCleaning(HoodieCleanMetadata
     LOG.info("Incremental Cleaning mode is enabled. Looking up partition-paths that have since changed "
         + "since last cleaned at " + cleanMetadata.getEarliestCommitToRetain()
         + ". New Instant to retain : " + newInstantToRetain);
+    String commitJustBeforeEarliestCommitToRetain = null;
```

Comment:
Just wondering if these changes in this method are actually needed. Even if we keep the original logic, I guess we are not missing out on any scenario. Please correct me if I am missing anything here. Ultimately you are only getting the lastCheckedCommit and that is already taken care of in the original logic.